### ISSUE TITLE
Support

### ISSUE THREAD

**Issue by binoculars88** [jleszczynski/cordova-vr-player#3]
Do you maybe have some info about the support of this plugin?
VR view native supports:

- iOS 8 and higher.
- Android 4.4 (Kit Kat) and higher.

Is this the same for this plugin?
Thanks!

**Comment by georgski** [comment]
Yes, the plugin simply wraps the functionality of the native VR View, so I believe it should support the same platform versions as the Google VR SDK.
